src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/OpenTelemetryProtocolExporterEventSource.cs

```diff
@@ -44,16 +44,10 @@ public void FailedToReachCollector(Exception ex)
             }
         }
 
-        [Event(1, Message = "Exporter failed to convert SpanData content into gRPC proto definition. Data will not be sent. Exception: {0}", Level = EventLevel.Error)]
-        public void FailedToConvertToProtoDefinitionError(string ex)
+        [Event(3, Message = "Could not translate activity from class '{0}' and method '{1}', span will not be recorded.", Level = EventLevel.Informational)]
```

Comment:
The ordering is not corret. 1,2,3 events in that order would make it easy to maintain.

Reply:
the only thing is that i changed from public to private, so people wont use it incorrectly...should i revert it?

Reply:
just updated back. i think i did ths with other event sources. i will double-heck